docs/admin/tutorial.md

```diff
@@ -260,7 +215,15 @@ sudo systemctl stop bpfd.service
 
 ### Step 4-6
 
-Same as above except `sudo` can be dropped from all the `bpfctl` commands.
+Same as above except `sudo` can be dropped from all the `bpfctl` commands and `bpfctl` is now in $PATH:
+
+```console
+bpfctl load --path /home/dave/dev/xdp-tutorial/basic01-xdp-pass/xdp_pass_kern.o --section-name "xdp" xdp --iface vethb2795c7 --priority 100
+
+bpfctl list
+
+bpfctl unload 92e3e14c-0400-4a20-be2d-f701af21873c
```

Comment:
This UUID kind of comes from nowhere, maybe show the STDout from the above load/ list command here?

Reply:
It's the same UUID as on line 134, just dropped the `sudo ./target/debug/` part of the command. I'll cut and paste all the output from above here to make it clear.

Reply:
Done